pkg/cmd/backuprestore/backupserver.go

```diff
@@ -61,7 +88,43 @@ func (b *backupServer) Run(ctx context.Context) error {
 	defer cancel()
```

Comment:
NotifyContext returns a context and cancelFunc, you don't need the above WithCancel call.
https://pkg.go.dev/os/signal#NotifyContext

Reply:
done 👍🏽